views/container-statuses.html

```diff
@@ -0,0 +1,33 @@
+<div ng-if="containerStatuses.length == 0"><em>none</em></div>
+<dl ng-repeat="containerStatus in containerStatuses | orderBy:'name'" class="dl-horizontal">
+  <dt>Name</dt>
+  <dd>{{containerStatus.name}}</dd>
+  <dt>State</dt>
+  <dd>
+    <span ng-if="containerStatus.state | isEmpty"><em>none</em></span>
+    <span ng-repeat="(state, stateDescription) in containerStatus.state">
+      {{state}}
+      <span ng-if="stateDescription.startedAt">({{stateDescription.startedAt}})</span>
+      <span ng-if="!$last">,</span>
+    </span>
+  </dd>
+  <dt>Last State</dt>
+  <dd>
+    <span ng-if="containerStatus.lastState | isEmpty"><em>none</em></span>
+    <span ng-repeat="(state, stateDescription) in containerStatus.lastState">
+      {{state}}
+      <span ng-if="stateDescription.startedAt">({{stateDescription.startedAt}})</span>
+      <span ng-if="!$last">,</span>
+    </span>
+  </dd>
+  <dt>Ready</dt>
+  <dd>{{containerStatus.ready}}</dd>
+  <dt>Restart Count</dt>
+  <dd>{{containerStatus.restartCount}}</dd>
+  <dt>Image</dt>
+  <dd>{{containerStatus.image}}</dd>
```

Comment:
not sure we need to show image/imageID/containerID here... takes up a lot of space and should be the same as in the pod spec